src/testdir/test_popup.vim

```diff
@@ -1830,4 +1830,28 @@ func Test_popup_completion_many_ctrlp()
   bw!
 endfunc
 
+func Test_pum_reverse_render()
+  CheckScreendump
+  let lines =<< trim END
+    set wildmenu
+    set wildoptions=pum,reverse
+    set cot+=noinsert,reverse
+    set pumheight=5
+    call setline(1, ['apple', 'anchor', 'astronomy', 'artist', 'avenue', 'antique', 'adventure', 'arrival', 'amber', 'alliance'])
+  END
+  call writefile(lines, 'Xscript', 'D')
+  let buf = RunVimInTerminal('-S Xscript', {})
+
+  call term_sendkeys(buf, ":set lines=13\<CR>")
```

Comment:
Does this need to be set after Vim's startup? Can you set `rows` in the options for `RunVimInTerminal()` instead?

Reply:
aha i am missing rows ..